FirebaseAuth/Sources/Public/FirebaseAuth/FIRAuth.h

```diff
@@ -88,7 +87,10 @@ typedef void (^FIRAuthDataResultCallback)(FIRAuthDataResult *_Nullable authResul
                                           NSError *_Nullable error)
     NS_SWIFT_NAME(AuthDataResultCallback);
 
-#if defined(__IPHONE_10_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0
+#if (defined(__IPHONE_10_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_10_0) || \
```

Comment:
I believe this is only here to support building on Xcode 7 and earlier. And can be deleted.

Reply:
Thank god